packages/tasks/src/tasks/text-to-speech/data.ts

```diff
@@ -40,7 +40,11 @@ const taskData: TaskDataCustom = {
 		},
```

Comment:
We can replace `suno/bark` with https://huggingface.co/parler-tts/parler-tts-large-v1